src/main/java/com/jfrog/ide/idea/ui/LocalComponentsTree.java

```diff
@@ -30,6 +29,7 @@
  */
 public class LocalComponentsTree extends ComponentsTree {
     private static final String SHOW_IN_PROJECT_DESCRIPTOR = "Show direct dependency in project descriptor";
+    private static final String IGNORE_RULE_TOOL_TIP = "Create ignore rule option is only available if JFrog Project or watches are defined";
```

Comment:
I think this tooltip doesn't appear when the button is disabled
```suggestion
    private static final String IGNORE_RULE_TOOL_TIP = "Create ignore rule option is only available if JFrog Project or Watches are defined";
```